### DISCUSSION TITLE
Is it possible to open a PTY in the center of a a terminal?

### DISCUSSION THREAD

**Discussion by sweetbbak** [creack/pty#187] [Q&A]
Is it possible to open a PTY in the center of a a terminal or to somehow set its X or Y offset?

**Comment by creack** [comment]
Not that I know of. Not sure how tmux/screen achieve this. If you find the info, I would be curious.

I suspect it doesn't have to do directly with the pty as the pty still thinks the origin 0,0 is where the split starts.
